src/mpid/ch4/shm/posix/posix_noinline.h

```diff
@@ -13,6 +13,10 @@ int MPIDI_POSIX_mpi_init_hook(int rank, int size, int *n_vcis_provided, int *tag
 int MPIDI_POSIX_mpi_finalize_hook(void);
 int MPIDI_POSIX_get_vci_attr(int vci);
 
+int MPIDI_POSIX_coll_cvars_init(void);
```

Comment:
Are these functions used anywhere other than `posix_init.c`. If not, they are essentially internal functions for `posix_init.c`, and should be not have a `MPIDI_POSIX_` prefix.